static/templates/admin/runEnhancementsForm.html

```diff
@@ -0,0 +1,19 @@
+<form id="run_enhancements_form"class="edit_form">
+	<p>
+		All appropriate enhancements will be performed on the selected objects and their children
```

Comment:
If we did so, we would likely want to change the title of the window to say that the children of the object would also get enhancements run.  The window looks slightly sad with nothing in it but the checkbox :(